### ISSUE TITLE
It's unclear which loss and metrics are used for training and evaluation

### ISSUE THREAD

**Issue by mnmjh1215** [samgregoost/atmg#1]
First of all, thanks for your remarkable work.

I wanted to study your method, ATMG, in depth, but it is not clear which loss the ATMG model uses (https://github.com/samgregoost/atmg/blob/master/atmg/models.py#L331-L389) and which metric is used for evaluation (https://github.com/samgregoost/atmg/blob/master/atmg/evaluation/classification.py#L134-L141) as many lines are commented out.

Would it be possible if you could make code more clear to so that the loss and metric used for ATMG is more distinguishable from those of MERU?

Thanks.

**Comment by samgregoost** [comment]
Thank you for pointing this out. We have improved our code based on your comment.

**Comment by mnmjh1215** [comment]
Thanks for the update!
So it seems that the einstein loss is not used for computing the loss. Is my understanding correct?

Also, the contrastive objective seems to be different from the one described in the paper, as the code use -ext(text_feats, image_feats) and ext(image_feats, text_feats) for computing the logits for contrastive loss while the contrastive objective described in the paper use -ext(text_feats, image_feats) and ext(text_feats, image_feats), where ext(a, b) is the pairwise exterior angle.
